src/BasketBundle/Controller/BasketController.php

```diff
@@ -35,6 +36,19 @@
  */
 class BasketController extends Controller
 {
+    /**
+     * @var BasketFactoryInterface
+     */
+    private $basketFactory;
+
+    /**
+     * @param BasketFactoryInterface $basketFactory A Sonata ecommerce basket interface
```

Comment:
Can be removed

Reply:
Hi i'm confused, one time i need to use typehint one time i should remove. i really don't get the needed. i reproduced same as constructor found here https://github.com/sonata-project/ecommerce/blob/master/src/BasketBundle/Controller/Api/BasketController.php

this enhancement is mandatory when using sf 4.1, we are not able to use this king of code 
`$this->get('sonata.basket.factory')->save($basket);`

The alias is dynamically create in sonataBasketExtension.

Reply:
The typehint must stay, the docblock should be removed

Reply:
So i should remove  ` @var BasketFactoryInterface` and  `@param BasketFactoryInterface $basketFactory A Sonata ecommerce basket interface`  ?

Reply:
Only the `@param.... `, because it does not add any value. its clear by the typehint

Reply:
Okay, done